api/sip/__init__.py

```diff
@@ -120,6 +120,7 @@ class SIP2Settings(BasicAuthProviderSettings):
             options={
                 Sip2Dialect.GENERIC_ILS: "Generic ILS",
                 Sip2Dialect.AG_VERSO: "Auto-Graphics VERSO",
+                Sip2Dialect.TZ_SPACES: "TZ Spaces",
```

Comment:
I know there was some discussion of this in the ticket, I think this would be better named `Folio` then `TZ Spaces`. I pinged Carissa and Courtney on the ticket for some feedback from their perspective.